static/src/javascripts/boot-webpack.js

```diff
@@ -31,8 +39,7 @@ domready(() => {
         }
 
         userTiming.mark('commercial request');
-        require(
-            ['bootstraps/commercial'],
+        import('bootstraps/commercial').then(
             raven.wrap({ tags: { feature: 'commercial' } }, commercial => {
```

Comment:
Just thinking out loud here. Does `raven.wrap` return whatever the enclosed function expression returns? If yes, then you can reduce to (simplifying):

```
import(...)
.then(raven.wrap(commercial => commercial.init())
.then(() => isEnhanced ? import(...) : () => ())
.then(bootEnhanced => bootEnhanced())
```